machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -173,7 +173,10 @@ export const ExistingMosipVCItemMachine =
                   actions: ['setMaxDownloadCount', 'setDownloadInterval'],
                 },
                 onError: {
-                  actions: log((_, event) => (event.data as Error).message),
+                  actions: [
+                    log((_, event) => (event.data as Error).message),
+                    'sendDownloadLimitExpireToVcMachine',
```

Comment:
why do we have "Machine" in state name?
we can use "sendDownloadLimitExpire"

Reply:
It is not an state, it was just an action that sends the event from ExistingMosipVCItemMachine.ts to Vc.ts. I will change the naming convention